examples/private_set_intersection/module/rust/src/lib.rs

```diff
@@ -46,27 +45,51 @@ oak::entrypoint!(oak_main => |_in_channel| {
     oak::run_event_loop(dispatcher, grpc_channel);
 });
 
+/// Maximum number of contributed private sets.
+pub const CONTRIBUTION_THRESHOLD: u64 = 2;
```

Comment:
This and the `set_count` field should probably be named similarly, since they represent the same thing.